.github/workflows/depcheck.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: ['16.x']
```

Comment:
_question_

Why not add node 20 here?

Reply:
I would like to attack Node.js 20 in another change.

Reply:
I can't tell, what is your opinion on running the cycles and depcheck in multiple versions of Node?

Reply:
I agree that we only need to run these checks in one version of Node.js and we could drop the matrix.